### DISCUSSION TITLE
Incident with Dispatch from CAD and Incident with Dispatch without CAD?

### DISCUSSION THREAD

**Discussion by adamjack** [ulfsri/neris-framework#16] [Q&A]
I am attempting to understand how incidents are intended to work with dispatches both with CAD and without CAD via the API. Does CAD create the dispatch at NERIS and somehow the API user gets notified, or does CAD notify both NERIS and the API user and the API user then queries NERIS for a dispatch to embed into the incident?

I suspect there are a few scenarios for the foreseeable future; no CAD, NERIS-enabled CAD, NERIS-not-enabled CAD (but maybe the first and last are effectively the same.)

Thanks in advance for guidance on this flow.

**Comment by msudol** [comment]
I may not be the best person on the team to answer this, but I didn't want to leave this question hanging for too long, because it's one we've got on our board that still requires a little work and planning on.  To start out, we're trying to get an idea of the numbers, what percentage of implementations will fall into these categories?  

Chronologically one would think, when CAD exists, it would initially create the incident. In practice it seems the existing RMS do much of the heavy lifting and already integrate or harmonize with CAD.  **My opinion** is that to avoid chicken/egg conditions or messy find/edit/complete workflows, that the best scenario would be that RMS create incidents in as many scenarios as possible.  

The only real "messy scenario" is if CAD is NERIS-enabled and RMS is also NERIS-enabled, but the 2 systems are not partnered in anyway. We are certainly open to suggestions.  I think there will need to be a feature to find/resolve duplicates or matches at some point but we're not quite there yet. 

Please Note: This is just my general musing on the topic and doesn't represent an official NERIS statement. Just thought this was an important question to get some discussion started.

**Comment by adamjack** [comment]
Matthew,

Are you asking how many departments have a Fire RMS which is not tightly
(two way or all details) integrated to CAD? I would suspect a lot. There is
a glorious tapestry of "technical creativity" out there. :-)

Even when both CAD and RMS become NERIS enabled, unless there is an NERIS
facilitated integration, the chances of full NERIS dispatch data being
communicated between the two seems low for many integrations for the
foreseeable future.

CAD/PSAP has the first access to the incident, and often dictates the
incident identifier (within CAD scope) but then (as you say) it then often
hands off to an RMS. That said, the apparatus times (often via radio to
dispatcher, for those without MDC) and PSAP call details are CAD data. I
would think the CAD would create a dispatch record (and maintain it) and an
RMS would reference it from the incident. With that, NERIS facilitated the
integration without CAD and Fire RMS having to coordinate much more than
some identifier.

regards,

Adam

On Mon, Aug 19, 2024 at 9:57 AM Matthew Sudol ***@***.***>
wrote:

> I may not be the best person on the team to answer this, but I didn't want
> to leave this question hanging for too long, because it's one we've got on
> our board that still requires a little work and planning on. To start out,
> we're trying to get an idea of the numbers, what percentage of
> implementations will fall into these categories?
>
> Chronologically one would think, when CAD exists, it would initially
> create the incident. In practice it seems the existing RMS do much of the
> heavy lifting and already integrate or harmonize with CAD. *My opinion*
> is that to avoid chicken/egg conditions or messy find/edit/complete
> workflows, that the best scenario would be that RMS create incidents in as
> many scenarios as possible.
>
> The only real "messy scenario" is if CAD is NERIS-enabled and RMS is also
> NERIS-enabled, but the 2 systems are not partnered in anyway. We are
> certainly open to suggestions. I think there will need to be a feature to
> find/resolve duplicates or matches at some point but we're not quite there
> yet.
>
> Please Note: This is just my general musing on the topic and doesn't
> represent an official NERIS statement. Just thought this was an important
> question to get some discussion started.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ulfsri/neris-framework/discussions/16#discussioncomment-10385882>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAZY7UZFZN5YVGERP2GXJ3ZSIIWFAVCNFSM6AAAAABLWVMVJWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMZYGU4DQMQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Adam R. B. Jack
https://www.neukadye.com
https://adamjack.wordpress.com/
